packages/trader/src/AppV2/Components/RiskManagementItem/risk-management-item.tsx

```diff
@@ -105,7 +106,7 @@ const RiskManagementItem = observer(
                     {!is_deal_cancellation &&
                         (is_accumulator ? (
                             <Text size='sm'>
-                                {finalValue} {currency}
+                                {FormatUtils.formatMoney(finalValue)} {currency}
```

Comment:
Will this work for other currencies with higher decimal places like btc, eth?

Reply:
It is changed into formatMoney function from the shared package for now , thanks